gflownet.py

```diff
@@ -375,20 +377,21 @@ def __init__(self, args, comet=None, proxy=None, al_iter=-1, data_path=None):
         self.no_log_times = args.gflownet.no_log_times
         # Make train and test sets
         self.buffer.make_train_test(
-            data_path, args.gflownet.train.path, args.gflownet.test.path, self.oracle
-        )
-        self.test_period = args.gflownet.test.period
-        self.test_score = args.gflownet.test.score
+            data_path, args.gflownet.train.path, args.gflownet.test.path, self.oracle, args = args
+        )#data_path='datasets/' + self.config.dataset.oracle + '.npy' 
+   
+        self.test_period = args.gflownet.test.period 
+        self.test_score = args.gflownet.test.score 
         if self.test_period in [None, -1]:
             self.test_period = np.inf
         # Train set statistics
-        if self.buffer.train is not None:
-            min_energies_tr = self.buffer.train["energies"].min()
-            max_energies_tr = self.buffer.train["energies"].max()
-            mean_energies_tr = self.buffer.train["energies"].mean()
-            std_energies_tr = self.buffer.train["energies"].std()
+        if self.buffer.train is not None: 
+            min_energies_tr = self.buffer.train[self.test_score].min()#"energies"
+            max_energies_tr = self.buffer.train[self.test_score].max()
+            mean_energies_tr = self.buffer.train[self.test_score].mean()
+            std_energies_tr = self.buffer.train[self.test_score].std()
```

Comment:
Please revert to "energies". If it fails because it's got a different name, we should change it in the file. Let me know if this gives your problems.